packages/in_app_purchase/in_app_purchase/README.md

```diff
@@ -65,29 +64,6 @@ This section has examples of code for the following tasks:
 * [Accessing platform specific product or purchase properties](#accessing-platform-specific-product-or-purchase-properties)
 * [Presenting a code redemption sheet (iOS 14)](#presenting-a-code-redemption-sheet-ios-14)
 
-### Initializing the plugin
-
-The following initialization code is required for Google Play:
-
-```dart
-// Import `in_app_purchase_android.dart` to be able to access the
-// `InAppPurchaseAndroidPlatformAddition` class.
-import 'package:in_app_purchase_android/in_app_purchase_android.dart';
-import 'package:flutter/foundation.dart';
-
-void main() {
-  // Inform the plugin that this app supports pending purchases on Android.
-  // An error will occur on Android if you access the plugin `instance`
-  // without this call.
-  if (defaultTargetPlatform == TargetPlatform.android) {
-    InAppPurchaseAndroidPlatformAddition.enablePendingPurchases();
-  }
-  runApp(MyApp());
-}
-```
-
-**Note:** It is not necessary to depend on `com.android.billingclient:billing` in your own app's `android/app/build.gradle` file. If you choose to do so know that conflicts might occur.
```

Comment:
We shouldn't remove this note; it's unrelated to the section above, and it was added based on client feedback.

Reply:
thank you for the feedback!